### DISCUSSION TITLE
Launch Event AMA

### DISCUSSION THREAD

**Discussion by endigo9740** [skeletonlabs/skeleton#1002] [Q&A]
Details on the launch day event can be found here:
- https://github.com/skeletonlabs/skeleton/discussions/1087

Hey everyone, as we get closer to v1.0 for Skeleton we would like to begin gathering questions from you, our community, related to the project. If there's something you're curious about or would like to ask us about the long term goals for the project, this is the place to do it. We'll host a launch day livestream event with a Q&A session and dive into your posts here. Additionally, due to the interactive nature, we may be able to demonstrate how to solve some of the issues raised, or just generally talk through our thoughts on any particular subject.

Anything goes, though we recommend you keep your questions tangentially related to Skeleton!

Thanks!

**Comment by riziles** [comment]
Do you know of any good resources with tips or tools to help facilitate migrating a project or package (especially one built on Sass or Less) to Tailwind? In my limited experience thus far, the process seems brutally arduous and time consuming.

**Comment by dirk-bester** [comment]
Congrats on almost 1.0!

**Comment by paoramati** [comment]
Will you one day add dedicated input and form components?

**Comment by endigo9740** [comment]
Great questions from Stew on Discord:

![Screenshot 2023-03-10 at 10 31 54 AM](https://user-images.githubusercontent.com/1509726/224370908-5b80b989-53c1-4eaa-80a1-8881fd2ac0de.png)
